check.go

```diff
@@ -379,23 +383,23 @@ func (c *CheckRunner) handleCheckUpdate(check *api.HealthCheck, status, output s
 	metrics.IncrCounter([]string{"check", "txn"}, 1)
 	ok, resp, _, err := c.client.Txn().Txn(ops, nil)
 	if err != nil {
-		c.logger.Printf("[WARN] Error updating check status in Consul: %v", err)
+		c.logger.Warn("Error updating check status in Consul: ", "error", hclog.Fmt("%v", err))
 		return
 	}
 	if len(resp.Errors) > 0 {
 		var errs error
 		for _, e := range resp.Errors {
 			errs = multierror.Append(errs, errors.New(e.What))
 		}
-		c.logger.Printf("[WARN] Error(s) returned from txn when updating check status in Consul: %v", errs)
+		c.logger.Warn("Error(s) returned from txn when updating check status in Consul: ", "error", hclog.Fmt("%v", errs))
 		return
 	}
 	if !ok {
-		c.logger.Printf("[WARN] Failed to atomically update check status in Consul")
+		c.logger.Warn("Failed to atomically update check status in Consul")
 		return
 	}
 
-	c.logger.Printf("[TRACE] Registered check status to the catalog with ID %v", strings.TrimPrefix(string(check.CheckID), check.Node+"/"))
+	c.logger.Trace("Registered check status to the catalog with ID ", "CheckId", hclog.Fmt("%v", strings.TrimPrefix(string(check.CheckID), check.Node+"/")))
```

Comment:
Would you mind removing the extra whitespace after "ID"? Would you also mind making the key in camel-case i.e. "checkId"?

Reply:
Can you make "CheckId" lowercase? e.g. "checkId"